### DISCUSSION TITLE
Can we apply for Campus Expert while pursuing online degree

### DISCUSSION THREAD

**Discussion by tarunjain1st** [community/community#78097] [GitHub Education]
### Hi everyone,

I'm currently doing my MS in Computer Science online from Woolf University, 
* can i apply to campus expert program if enrolled to online course?
* if yes, do we required college email for enrolling?

**Comment by IgnacioPalma** [comment]
Hello @tarunjain1st! :wave: 

Campus Expert here:)
The requirements to apply to the program Campus Experts are the following:
- Be at least 18+ years of age.
- Validate your student status through the [GitHub Student Developer Pack](https://education.github.com/pack).
- Be enrolled in a post-secondary formal education institution.
- Have more than one year left as a student before graduating.
- Be a GitHub user for at least six months.
- Not be enrolled in the GitHub Campus Advisors Program.

If you can validate your student status through the GitHub Student Developer Pack via Woolf University, then you be able to apply for the program:)
Hope this helps!

PD: The next applications open in February :heart:

**Reply by tarunjain1st** [reply]
Thanks for help @IgnacioPalma
But unfortunately, i didn't got college email id, any other way we can get this

**Reply by IgnacioPalma** [reply]
If you're unable to verify your enrollment in a post-secondary formal education institution for requesting the Student Developer Pack, then I'm afraid that you don't meet the requirements for applying for the Campus Expert program:(

**Reply by tarunjain1st** [reply]
Okay, will ask institute incharge to look into it.
also can you mark this post as answered, i just need to show to my mentor.
